tasks/configure-OpenBSD.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+# XXX rpc.lockd(8) of OpenBSD does not support client-side of the protocol
```

Comment:
nothing else but just i could not understand why the comment is written here without reading other configure-*.yml.